packages/shared/src/contexts/GiveAwardModalContext.tsx

```diff
@@ -19,20 +21,12 @@ const AWARD_TYPES = {
 } as const;
 export type AwardTypes = keyof typeof AWARD_TYPES;
 
-const SCREENS = {
```

Comment:
Duplicate. We have another one which contains more items.

Reply:
I am not sure if it is duplicate, since different context should have limited number of different screens, at least thats how Chris and I imagined it?

Reply:
Oh. Thought they were duplicates. I can return it back, maybe I'll adjust the naming to avoid confusion.

Reply:
yeah that would work

Reply:
Done! 🚀